sdk/eventhub/event-processor-host/package.json

```diff
@@ -58,7 +58,7 @@
     "unit-test": "npm run unit-test:node && npm run unit-test:browser"
   },
   "dependencies": {
-    "@azure/event-hubs": "^2.1.1",
+    "@azure/event-hubs": "5.0.0-preview.3",
```

Comment:
Why is this change being made? EPH will be using the older version of event hubs and is not compatible with version 5